client/homescreen/activity-panel/style.scss

```diff
@@ -0,0 +1,73 @@
+.woocommerce-activity-panel-card {
+	.components-panel__body-title {
+		margin: 0;
+		padding: 1em 0;
+		.components-button {
+			display: flex;
+			flex-direction: row;
+			width: 100%;
+			padding-left: 24px;
+			font-style: normal;
+			font-weight: 400;
+			font-size: 20px;
+			line-height: 28px;
+			.woocommerce-activity-panel-header {
+				order: 1;
+				text-align: left;
+				width: 50%;
+				.woocommerce-activity-panel-title {
+					vertical-align: bottom;
+				}
+			}
+			& > span {
+				order: 2;
+				width: 50%;
+				text-align: right;
+				.components-panel__arrow {
+					width: 30px;
+					height: 30px;
+					vertical-align: text-top;
+				}
+			}
+		}
+	}
+
+	.woocommerce-activity-panel-badge {
+		background-color: $gray-100;
+		border-radius: 20px;
+		display: inline-block;
+		text-align: center;
+		font-style: normal;
+		font-weight: 600;
+		font-size: 14px;
+		line-height: 27px;
+		align-items: center;
+		width: 32px;
+		height: 28px;
+		margin-left: $gap;
+	}
+
+	.woocommerce-activity-card__actions {
+		margin: 0;
+		.components-button {
+			background: $gray-100;
+			border-radius: 12px;
+			height: 24px;
+			padding-left: $gap;
+			width: 65px;
+		}
+	}
+
+	.woocommerce-activity-card {
+		padding: ( $fallback-gutter / 2 ) $gutter;
+		&__header {
+			margin-bottom: $gap-smaller;
+		}
+	}
+}
+
+.components-panel__body-title {
```

Comment:
Any concerns with this not being scoped to the panel card? I could see this causing style pollution in other places. I realise that sometimes its necessary but I wanted to check.

Reply:
Good eyes! I fixed it in the commit 1dbcfec